### PR TITLE
[tests-only][full-ci]Adds spaces tests on `apiWebdavProperties1`

### LINKED ISSUE #3042: add spaces webdav for `apiWebdavProperties1` suite
### Description
- add spaces WebDAV URL for `apiWebdavProperties1` suite from `oc/core`
- includes:
  - copyFile.feature
  - createFileFolderWhenSharesExist.feature
  - createFolder.feature
  - getQuota.feature
  - setFileProperties.feature

### Related
- part of https://github.com/owncloud/ocis/issues/2854

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
adds spaces DAV path on `apiWebdavProperties1` suite

## Related Issue

- fixes https://github.com/owncloud/ocis/issues/3042

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require an increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by saw-jan — COMMENTED** [review]
LGTM

File: tests/acceptance/features/apiWebdavProperties1/setFileProperties.feature [hunk 1]
```diff
@@ -76,11 +96,10 @@ Feature: set file properties
...
       | new                | old               |
 
-  @issue-ocis-reva-276
-  Scenario: Setting custom DAV property using an old endpoint and reading it using a new endpoint
-    Given using old DAV path
-    And user "Alice" has uploaded file "filesForUpload/textfile.txt" to "/testoldnew.txt"
-    And user "Alice" has set property "very-custom-prop" with namespace "x1='http://whatever.org/ns'" of file "/testoldnew.txt" to "constant"
-    And using new DAV path
-    When user "Alice" gets a custom property "very-custom-prop" with namespace "x1='http://whatever.org/ns'" of file "/testoldnew.txt"
-    Then the response should contain a custom "very-custom-prop" property with namespace "x1='http://whatever.org/ns'" and value "constant"
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiWebdavProperties1/setFileProperties.feature:86]: why remove this?

Comment (amrita-shrestha) [reply tests/acceptance/features/apiWebdavProperties1/setFileProperties.feature:86]: this scenario and above scenario is same .

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by phil-davis — APPROVED** [review]

**Pull request merged**